db/models.py

```diff
@@ -1 +1,38 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The 'bonus' field in the Skill model is defined as a CharField with max_length=255. If the bonus is intended to be a numerical value or a specific type of bonus, consider using a more appropriate field type, such as IntegerField or a custom choice field, to better represent the data.